assets/components/headers/countrySwitcherHeader/countrySwitcherHeader.jsx

```diff
@@ -8,10 +8,10 @@ import SvgGuardianLogo from 'components/svgs/guardianLogo';
 import CountryGroupSwitcher from 'components/countryGroupSwitcher/countryGroupSwitcher';
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 
-type PropTypes = {
+export type PropTypes = {
```

Comment:
Since now we are using the export/import of types together, why not to put the export of this type at the bottom of the file?

Reply:
You don't appear to be able to combine type exports with value exports as you can with imports. Also, this file has a default export.